readme.md

```diff
@@ -155,6 +155,9 @@ All the calls have async variants:
 
 ```csharp
 var response = client.IndexAsync(tweet, idx => idx.Index("mytweetindex")); // returns a Task<IndexResponse>
+
+// Or, in an async-context
+var response = await client.IndexAsync(tweet, idx => idx.Index("mytweetindex")); // unwraps a Task<IndexResponse>
```

Comment:
not sure I like the `unwrap` here.  That term is also used in TPL to go from `Task<Task<T>>` to just `Task<T>`. 
maybe change `unwrap` with `await`.

```suggestion
var response = await client.IndexAsync(tweet, idx => idx.Index("mytweetindex")); // awaits a Task<IndexResponse>
```

Reply:
Thanks, yep, you are right, `await` sounds suitable here.